Android.mk

```diff
@@ -3,6 +3,9 @@ LOCAL_PATH := $(call my-dir)
 VERSION = $(shell git describe --always --dirty=-dev 2>/dev/null || echo Unknown)
 OPTEE_CLIENT_PATH ?= $(LOCAL_PATH)/../optee_client
 
+# Not sure why this isn't being set for me.
+TA_DEV_KIT_DIR := optee/optee_os/out/arm-plat-hikey/export-ta_arm64
```

Comment:
The proper fix is probably going to depend on having the rest of optee built by android makefiles. Meanwhile, without this, it won't build. Maybe it can be set in a device file?